src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task2.java

```diff
@@ -0,0 +1,23 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a number please: ");
+        int startRange = scanner.nextInt();
+        int endRange = scanner.nextInt();
+        for (int i = startRange; i <= endRange; i++) {  //*declare the type of variable for i always FOR "for" loop
+            if (i % 3 == 0) {                          //* each if else statement has a "{}"
```

Comment:
Please, start with the most complex condition to avoid some results that are expected. (eg. 15 )

Reply:
Done